src/lightbox/Lightbox.js

```diff
@@ -69,11 +51,40 @@ export default function Lightbox(props: Props): Node {
       ? `Shared in #${streamNameOfStreamMessage(message)}`
       : 'Shared with you';
 
-  // Since we're using `Dimensions.get` (below), we'll want a rerender
-  // when the orientation changes. No need to store the value.
-  useGlobalSelector(state => getGlobalSession(state).orientation);
```

Comment:
Hmm, rereading this old code, I have a bit of a suspicion it might have a race: it seems possible we might rerender the component at a time when the value we get from `Dimensions.get` is different from the latest value Redux knows about.

If so, then we can likely get stuck in a confused state (stuck until the user rotates again):
* Say the user rotates three times in quick succession: P→L→P→L.
* We render at a time when two flips have happened, but only one has shown up in Redux. As a result, we render for P again, while Redux thinks we rendered for L.
* Then the last two flips reach Redux. It sees the state is L ­— and that's the same as it was on the last render, so it doesn't schedule a new render.
* We persist with a P render indefinitely, even though the real orientation is now L.

Anyway. Another reason to be glad to remove this!